lib/customers.js

```diff
@@ -514,10 +515,11 @@ function getCustomersList (phone = null, name = null, address = null, id = null)
       ) cn ON c.id = cn.customer_id
     WHERE c.id != $2
   ) AS cl WHERE rn = 1
+  AND enabled
```

Comment:
Can you use `c.enabled` instead? Just so that it's obvious where `enabled` comes from.